.github/workflows/ci.yml

```
@@ -0,0 +1,26 @@
+name: ci
```

Minor suggestion for a more descriptive name
```suggestion
name: CI workflow for Java and Gradle build
```